README.md

```diff
@@ -123,20 +123,97 @@ The CSV File contains the following output:
 | success | Whether or not the test was successful
 | expectedResponses | The possible expected response back from the utterance
 
-### DataDog
-DataDog captures metrics related to how all the tests have performed.
-
-The metrics can be easily reported on.
-
-They also can be used to setup notifcations when certain conditions are triggered.
-
-### Creating A Dashboard
-DataDog makes it easy to create a Dashboard.
-
-### Creating Alarms
-DataDog makes it easy to setup alarms.
+### **DataDog**
+DataDog captures metrics related to how all the tests have performed. Each time we run the tests, and when `datadog` has been set as the `metric` mechanism to use in the `config.json` file, we push the result of each test to DataDog.
+
+In this example, we are using below metrics:
+- `utterance.success`
+- `utterance.failure`
+
+The metrics can be easily reported on through a DataDog Dashboard (see next section).
+
+They also can be used to setup notifcations when certain conditions are triggered as shown below.
+
+### **Creating A Dashboard**
+DataDog makes it easy to create a Dashboard:
+- Click on Dashboards, then select "New Dashboard" from the left menu
+![Creating a DataDog dashboard step 1][DataDogCreatingDashboard1]
+- Give the Dashboard a name and select "New Timeboard" 
+![Creating a DataDog dashboard step 2][DataDogCreatingDashboard2]
```

Comment:
Use the html syntax for images:
```
<p align="center">
    <img src="images/DataDogCreatingDashboard1" height="300px">
</p>
```
Play with the height a bit, but ideally, the images would be about 1/3 to 1/2 of the page. Playing in the dev console, I think 250-300px works well